depthai_ros_driver/src/pipelines/multicam_pipeline.cpp

```diff
@@ -0,0 +1,77 @@
+#include <ros/console.h>
+
+#include <pluginlib/class_list_macros.h>
+
+#include <depthai_ros_driver/pipeline_ex.hpp>
+
+namespace depthai_ros_driver
+{
+class MulticamPipeline : public rr::PipelineEx {
```

Comment:
Could you please help me understand the reason for:
* Adding a pipelineEx (I assume, it's an example pipeline)
* Having a mobilenet pipeline as well as a mobilenet function in pipelineEx and multicam pipeline

Reply:
adding `pipelineEx` class is not necessary.
I did this because I could reuse the initilization code for color cam example and stereo example separately.
Current gen2 code is not stable and various conbination of streams or parameters don't work. I just wanted to keep examples of just a preview pipeline or just a stereo pipeline instead of implementing a huge custom pipeline.

Maybe in the end, I will delete pipelineEx, and just will keep some small example pipelines.

Reply:
Got it. Makes sense.